TechLadies/irrc-madwish
# IRRC-105. Increase pagination limit to display 20 rows per page

# Increase pagination limit to display 20 rows (students/teacher) per page 


[IRRC-105](https://bootcamp-irrc.atlassian.net/browse/IRRC-105)


## Description

Increase the number of rows of students and teachers that can be displayed on each page. It is currently set at a maximum of 20. 


## Test Steps

1. Create at least 20 new Students
2. Create at least 20 new Teachers
3. You should see a max of 20 new students / teachers on each page
